Source/ImGui/Private/ImGuiModule.cpp

```diff
@@ -261,7 +261,7 @@ void FImGuiModule::ToggleShowDemo()
 // Runtime loader
 //----------------------------------------------------------------------------------------------------
 
-#if !WITH_EDITOR && RUNTIME_LOADER_ENABLED
+#if WITH_EDITOR || RUNTIME_LOADER_ENABLED
```

Comment:
This change is preventing ImGUI from running in standalone mode. Reverting this change launches the standalone player.

Reply:
The existing `#if WITH_EDITOR || RUNTIME_LOADER_ENABLED` is correct. The issue is that the `RUNTIME_LOADER_ENABLED` define isn't being set to `1` in standalone mode.